fitbenchmarking/utils/misc.py

```diff
@@ -52,20 +52,18 @@ def get_minimizers(software_options):
         return minimizers, software
 
 
-def setup_fitting_problems(data_dir, group_name):
+def setup_fitting_problems(data_dir):
```

Comment:
Could you nuke this method altogether in the code base and instead just call get_problem_files directly?

Reply:
Yes, and done